include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -31,13 +31,15 @@
 #include <deal.II/dofs/dof_tools.h>
 
 #include <deal.II/fe/fe_tools.h>
+#include <deal.II/fe/fe_values.h>
```

Comment:
Do you really need this?

Reply:
We'll need it later on to compute real support points, so I removed it for the time being